### DISCUSSION TITLE
Gradlew commands not working

### DISCUSSION THREAD

**Discussion by OroArmor** [community/community#25526] [Actions]
<p>Im currently writing a library for my robotics team, and I want to use gradle as a way to automatically test the code.</p>
<p>I followed the directions on how to setup an action, and once I got gradle to properly work locally, I then wanted to get it to work on github.</p>
<p>Right now, Im getting this error:</p>
<pre><code>/home/runner/work/_temp/deaebf63-2b6d-4c98-b6b6-8c262b8231f8.sh: line 1: ./gradlew: Permission denied
##[error]Process completed with exit code 126.
</code></pre>
<p>The link to the yml file is: <a href="https://github.com/robototes/2412RobotLibrary/blob/OroArmor-gradletest/.github/workflows/gradle.yml" rel="nofollow noopener">https://github.com/robototes/2412RobotLibrary/blob/OroArmor-gradletest/.github/workflows/gradle.yml</a> </p>

**Comment by BrightRan** [comment, marked as answer]
<p>You can try to execute the following command before run gradle commands.</p>
<pre><code>chmod +x ./gradlew
</code></pre>
<p><img src="https://user-images.githubusercontent.com/46307996/181091724-26f618d4-3c19-4aac-85f0-8f30e30dc7a7.png" alt="workaround.png" data-base62-sha1="uVoyyzkhcqAXI3jL8DiHjz58Mow" width="232" height="103"></p>
<p>You also can reference this earlier reported ticket for the same error: <a href="https://github.com/actions/starter-workflows/issues/171" rel="nofollow noopener">https://github.com/actions/starter-workflows/issues/171</a></p>

**Reply by lewisc99** [reply]
thanks, it worked it

**Comment by OroArmor** [comment]
<p>Thank you! It worked perfectly!</p>
